app/javascript/components/StudyResults.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import Tab from 'react-bootstrap/lib/Tab'
+import Tabs from 'react-bootstrap/lib/Tabs'
+import { useTable, usePagination } from 'react-table'
+
+/**
+ * Wrapper component for studies tab on homepage
+ */
+const StudyResults = props => {
+  return (
+    <Tab.Container id="result-tabs" defaultActiveKey="study">
+      <Tabs defaultActiveKey='study' animation={false} >
+        <Tab eventKey='study' title="Studies" >
```

Comment:
maybe 'changePage' instead of turnPage, to make it clearer that it can go to any page?

Reply:
f049d0e1